api/discovery/v1/foreigncluster_types.go

```diff
@@ -53,11 +54,13 @@ type ForeignClusterStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	Joined             bool                `json:"joined"`
-	PeeringRequestName string              `json:"peering-request-name,omitempty"`
-	CaDataRef          *v1.ObjectReference `json:"caDataRef,omitempty"`
-	Advertisement      *v1.ObjectReference `json:"advertisement,omitempty"`
-	Ttl                int                 `json:"ttl,omitempty"`
+	OutgoingJoin             bool                `json:"outgoingJoin"`
```

Comment:
Proposal: can we have two distinct sections for outgoing and incoming? This way we can clearly understand the status of peering in each direction, and the overall parameters...